### DISCUSSION TITLE
Ability to clear localStorage landing/crash statistics

### DISCUSSION THREAD

**Discussion by lukeify** [ehmorris/lunar-lander#17] [Ideas]
👋 Would you consider a PR from me to clear `localStorage` of the game's statistics? 

I don't know how you'd prefer to take the UI direction of in-game controls, but for now, would hovering over the statistics in the top right corner and have it change to a label that reads "Clear" be acceptable?

**Comment by ehmorris** [comment]
Thanks for the suggestion. I'll think about doing this plus audio on/off somewhere.
